src/language-css/parser-postcss.js

```diff
@@ -298,6 +298,23 @@ function parseNestedCSS(node, options) {
       node.raws.value = selector;
     }
 
+    // Move comments that are stuck in between into value for sass only
+    if (
+      node.raws.value &&
+      node.raws.value.scss &&
```

Comment:
`isScssParser()`

Reply:
Please move it to utils and rename the function

Reply:
I named it isSCSSParserForNode, since there is already an isSCSSParser function